com.spotify.json

```diff
@@ -0,0 +1 @@
+{"name":"com.spotify","author":"hkini@thingpedia.stanford.edu","main":"index.js","thingpedia-version":38}
```

Comment:
You should add yourself as author too!

Also, you should remove the thingpedia-version field.

Reply:
will do...the original device was Hemanth's and I forgot to ever add my name.